certified-connectors/The IT Tipster/readme.md

```diff
@@ -0,0 +1,11 @@
+# The IT Tipster
```

Comment:
@theittipster1 please include all required sections in the readme file. Please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).